app/models/fansub.rb

```diff
@@ -11,6 +11,6 @@ class Fansub < ActiveRecord::Base
   }
 
   def current_release
-    self.releases.pending.order('id ASC').limit(1).first
+    self.releases.pending.sort_by { |release| release.source.number }.first
```

Comment:
I'd like this to be all ActiveRecord, but it's not clear how to do that right now. Additional investigation is needed